packages/react-native-win32/src/Libraries/Components/Touchable/Tests/TouchableWin32Test.tsx

```diff
@@ -424,23 +429,65 @@ class TouchableHighlightExample extends React.Component<{}, IExampleState> {
   };
 }
 
+const TouchableFocusExample = () => {
+  const [focused, setFocused] = React.useState(false);
+  let focusableRef = React.useRef<ViewWin32>(null);
```

Comment:
you need to use setAndForwardRef a la ViewWin32 (doesn't need to be a function component), this breaks using TouchableWin32 as a refobject